docs/docs/gatsby-style-guide.md

```diff
@@ -554,7 +559,7 @@ Here is how we measure the quality of the tutorial and docs. We will use ([cohor
 
 - time to value: how quickly, effectively, and painlessly can users experience the value of Gatsby?
 - daily, weekly, monthly active users
-- daily, weekly, monthly “finishers” of the tutorial (finisher = people who spend 5 mins minimum on each page of tutorial and go through whole thing)
+- daily, weekly, monthly “finishers” of the tutorial (finishers = people who spend 5 mins minimum on each page of tutorial and go through whole thing)
```

Comment:
Just a note, I think the way we measure this might evolve as we actually try to measure it.